client/blocks/reader-full-post/style.scss

```diff
@@ -629,61 +637,102 @@ $reader-full-post-desktop-min: $reader-full-post-sidebar-width + $reader-full-po
 	button.is-active,
 	.like-button.is-liked {
 		svg.reader-external path {
-			fill: var(--color-link);
+			fill: var( --color-link );
 		}
 
 		svg.reader-comment path,
 		svg.reader-share path {
-			stroke: var(--color-link);
+			stroke: var( --color-link );
 		}
 		svg.reader-star path {
-			stroke: var(--color-link);
+			stroke: var( --color-link );
 		}
 	}
 
 	.like-button.is-liked {
 		svg.reader-star path {
-			fill: var(--color-link);
+			fill: var( --color-link );
 		}
 	}
 }
 
 .wp-block-latest-posts__post-date::before {
-	content: " - ";
+	content: ' - ';
 }
 
 .is-reader-full-post {
-	background: var(--color-surface);
+	background: var( --color-surface );
 }
 
 .reader-full-post {
+	.reader-full-post__header--recent {
+		display: grid;
+		grid-template-columns: auto 1fr;
+		grid-template-rows: auto auto;
+		align-items: start;
+
+		.reader-full-post__header-site-icon {
+			grid-column: 1;
+			margin-right: 30px;
+			grid-row: 1 / 3;
+			align-self: center;
+			width: 80px;
+
+			.reader-full-post__site-icon {
+				border-radius: 50%;
+				height: 80px;
+				width: 80px;
+			}
+		}
+		.reader-full-post__header-title {
+			grid-column: 2;
+			grid-row: 1;
+		}
+
+		.reader-full-post__header-meta {
+			display: block;
+			grid-column: 2;
+			grid-row: 2;
+
+			span {
+				line-height: 1.6;
+				margin-right: 20px;
+				color: var( --color-text-subtle );
+				display: inline-block;
+			}
+			.reader-full-post__header-date {
+				margin-right: 0;
+			}
+		}
+	}
+
```

Comment:
These are the actual changes to this file. Prettier performed all the other changes, which are linting. I propose we go ahead and ship the lint updates with this PR.